contracts/EllipticCurve.sol

```diff
@@ -10,6 +10,9 @@ pragma solidity >=0.5.3 <0.7.0;
  */
 library EllipticCurve {
 
+  /// Pre-computed constant for 2 ** 255
+  uint256 constant private TWO_TO_THE_TWO_HUNDRED_AND_FIFTY_FIVE = 57896044618658097711785492504343953926634992332820282019728792003956564819968;
```

Comment:
```suggestion
  uint256 constant private U255_MAX_PLUS_1 = 57896044618658097711785492504343953926634992332820282019728792003956564819968;
```

Reply:
It's just a suggestion, I am not really convinced by this naming. :thinking:

Reply:
Why don't you just use `2**255`? If it's a constant it will be computed at compile time:
```suggestion
  uint256 constant private TWO_TO_THE_TWO_HUNDRED_AND_FIFTY_FIVE = 2**255;
```

Reply:
HAHAHA have you verified that argument @tmpolaczyk ? One thing I've learnt in the last years is to never make assumptions about the capabilities of `solc` :laughing: